deploy/rbd/kubernetes/v1.14+/csi-rbdplugin-provisioner.yaml

```diff
@@ -85,6 +85,20 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
+        - name: csi-resizer
+          image: quay.io/k8scsi/csi-resizer:v0.3.0
+          args:
+            - "--csi-address=$(ADDRESS)"
+            - "--v=5"
+            - "--csiTimeout=150s"
+            - "--leader-election"
```

Comment:
(unrelated to this patch) We now will have 4 side cars performing leader election, and hence each sidecar can determine in isolation that one of the many available controller plugin instance are the leader for the said operation. This defeats the controller locks against volumes if different operations will reach different controller plugins. This either needs to be understood better (IOW, the problem does not exist) or we may need to work with kubernetes CSI side cars to see how best to ensure all side car leaders land on the same instance.

Reply:
Sure.. this need more discussion.